packages/webapp/src/components/Filter/FilterGroup/index.jsx

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
The contents of this component are derived from part of what used to be in the `FilterPage` component. I abstracted this to a separate one so it can be reused both for `FilterPage` and for my modal/drawer